build/publish-latest.ts

```diff
@@ -17,6 +17,8 @@ export async function publishLatestToNpm() {
     'schematics',
     'eslint-plugin',
     'data',
+    'signals',
+    'operators',
```

Comment:
We can now rollback changes from [`config.ts`](https://github.com/ngrx/platform/pull/4002/files#diff-346737222179600781b6e608dd452f1f9e662081473dc1d0ad51bb6bf21250dbR24) and [`copy-schematics-core.ts`](https://github.com/ngrx/platform/pull/4002/files#diff-ed45f28768886664e000958eafce01133876762a7f7fb8d0a88f6cfe74e389b8R12)